### ISSUE TITLE
Selection of DNA seq for CDS parts

### ISSUE THREAD

**Issue by vinoo-igem** [iGEM-Engineering/iGEM-distribution#10]
We'll need to consider the process for choosing the DNA sequence of new CDS parts when one might not exist/be preferred for our purposes, and how to appropriately reference them. 

This might also span into a discussion on chassis and codon optimization vs harmonization. But for now, I think it would help to recommend a step-by-step process for navigating this particular issue.

**Comment by jakebeal** [comment]
I don't think that this is possible this year, but ultimately I think that every part that goes into the distribution should be validated through interlaboratory testing in an appropriate chassis organism. 

Right now, there are simply too many variables to be able to safely predict that something will work even with "simple" changes like codon optimization or shifting to a different assembly standard. There are simply too many opportunities for the biological equivalent of a syntax-breaking typo. We might consider whether this is an appropriate area to start to involve iGEM teams again as well, with a somewhat less rigid structure than the prior interlabs. 

In the interim, I think we can consider a sequence to be appropriate to include if it has been reported functional in any one of the following:
- the current iGEM registry
- a peer-reviewed publication
- data provided along with the submission

For purpose of the distribution, I personally would not trust any sequence that has even been just chassis or codon optimized away from the original. Even though these are fairly reliable processes, a mistake leading to a non-functional sequence could cause a lot of team frustration.

**Comment by jakebeal** [comment]
To close this issue, I think a recommended approach should be documented somewhere.
